CMakeLists.txt

```diff
@@ -64,6 +67,28 @@ check_include_file(strings.h HAVE_STRINGS_H)
 if( HAVE_STRINGS_H AND NOT HAVE_STRING_H )
   set(NEED_BSD_STRINGS 1)
 endif()
+if(IOS)
+  set(HAVE_UNSIGNED_CHAR 1)
```

Comment:
Actually it may be a valid CMake feature: check type exists without returning size.